lib/interactor/contracts.rb

```diff
@@ -164,7 +164,7 @@ def define_assurances_hook
 
         after do
           assurances = self.class.assurances.new
-          result = assurances.call(context)
+          result = assurances.call(context.to_h)
```

Comment:
Yes, when I started testing it it was raising a `NoMethodError: undefined method `key?' for`, it seems dry-validation started checking for keys using `#key?` method which doesn't exist in OpenStruct neither Context.